src/Compilers/Core/Portable/Operations/IBinaryOperatorExpression.cs

```diff
@@ -32,5 +32,13 @@ public interface IBinaryOperatorExpression : IHasOperatorMethodExpression
         /// value types.
         /// </summary>
         bool IsLifted { get; }
+        /// <summary>
+        /// <code>true</code> if this is a 'checked' binary operator.
+        /// </summary>
+        bool IsChecked { get; }
```

Comment:
I agree with Fred, this seems specific to expressions, and if we had an IExpression, it would logically move there, but it doesn't make sense for rest of operations (statements, query clauses, patterns, etc.)